### ISSUE TITLE
GS  online update openpli-4-mbmini-20151118 to openpli-4-mbmini-20151220 

### ISSUE THREAD

**Issue by gawro** [openmb/open-pli-core#2]
< 293.948336> [ePicLoad] setPara max-X=120 max-Y=40 aspect_ratio=0.000000 cache=0 resize=1 bg=#FF000000 auto_orient=0
 < 294.004430> [ePicLoad] decode picture... /usr/lib/enigma2/python/Plugins/Extensions/IPTVPlayer/icons/logos/anyfileslogo.png
 < 294.040292> [EXIF] getting exif from PNG
 < 294.253961> PC: 00507e1c
 < 294.254466> 00000000 00000001 8201541f 00000000
 < 294.254630> 7fffffff 02009028 00000000 02025bac
 < 294.254751> 000000b7 00000000 77dec000 00000000
 < 294.254878> 00000000 00000000 00000000 00507db4
 < 294.254998> 00000028 00000000 7fd9f6b8 00000000
 < 294.258264> 00000000 00000000 02015420 4ba7ae7f
 < 294.259216> 000013c9 77264cc0 00000000 00000000
 < 294.259497> 006557d0 7fd9f620 01e555b0 00507e08
 < 294.259659> -------FATAL SIGNAL
 ]]>

**Comment by gawro** [comment]
This problem occurred because the new soft no "Algorithmics / MIPS FPU emulator v1.5" Old openpli-4-mbmini-20151118 contains fpu emulatur Is it possible compilation of FPU emulator v1.5 because it is necessary to Openpera and IPTV  exteplayer3

 Regards

**Comment by openmb** [comment]
Could You make test using brand new flashed software ? If the problem appears ?

**Comment by gawro** [comment]
Yes using brand new flashed software error does not occur but the in the new software does not have 
"Algorithmics / MIPS FPU emulator v1.5"   Is it possible compilation new software of FPU emulator ?

Regards
